api/app/GraphQL/Mutations/PoolCandidateSnapshot.graphql

```diff
@@ -3,10 +3,6 @@
 query getProfile($userId: UUID!) {
   user(id: $userId) {
     id
-    authInfo {
```

Comment:
> We likely should not store sub in the snapshot for security reasons

This makes sense to me, but can we confirm this as it is not technically part of the issue's scope?